ui/app/(prowler)/new-overview/components/risk-severity-chart.tsx

```diff
@@ -0,0 +1,73 @@
+"use client";
+
+import { HorizontalBarChart } from "@/components/graphs/horizontal-bar-chart";
+import { BarDataPoint } from "@/components/graphs/types";
+import {
+  BaseCard,
+  CardContent,
+  CardHeader,
+  CardTitle,
+} from "@/components/shadcn";
+
+interface RiskSeverityChartProps {
```

Comment:
We should support only one level of interfaces


```suggestion
    interface RiskSeverityChartData {
     critical: number;
     high: number;
     medium: number;
     low: number;
     informational: number;
    }
    
    interface RiskSeverityChartProps {
      severityData: RiskSeverityChartData;
    }
```